charts/stable/wallos/Chart.yaml

```diff
@@ -6,7 +6,7 @@ annotations:
   truecharts.org/min_helm_version: "3.11"
   truecharts.org/train: stable
 apiVersion: v2
-appVersion: latest
+appVersion: 2.36.2
```

Comment:
Lets not change things that are automatically maintained by the tooling.

Reply:
if i did the change from latest to a semver, i didnt see a point in leaving it as latest value.

Reply:
reverted my changes if that was really necessary honestly.